apps/client/src/app/dashboard/page.tsx

```diff
@@ -4,6 +4,7 @@ import MaxWidthWrapper from "@/common/components/utils/max-width-wrapper";
 import DashboardHeader from "@/modules/dashboard/dashboard-header";
```

Comment:
### Critical Points:
❌ The PR introduces a new component `RepositoryName` without providing its implementation. This could lead to runtime errors if the component is not defined.

### Good Points:
👍 The PR removes the hardcoded "Repository Metrics / Overview" heading, making the dashboard more dynamic and potentially allowing for different repository names.
👍 The PR introduces a new component `RepositoryName` which could potentially improve the dashboard's user experience by displaying the repository name more prominently.